docker-compose.yml

```diff
@@ -5,4 +5,19 @@ services:
     image: php-server
     build: .
     ports:
-      - 8080:8080
\ No newline at end of file
+      - 8080:8080
+
+  mongodb:
+    image: mongo:latest
+    container_name: 'mongodb'
+    environment:
+      - MONGO_DATA_DIR=/data/db
+      - MONGO_LOG_DIR=/dev/null
+    volumes:
+      - mongodb_data:/var/lib/mongodb
+    ports:
+      - 27017:27017
+
+volumes:
+  mongodb_data:
```

Comment:
What is mongodb_data volume?

Reply:
this is the volume we want to map do the mongodb service

Reply:
But what location is it in the container? And you set MONGO_DATA_DIR=/data/db and there's no use to that volume.